protected/humhub/libs/SelfTest.php

```diff
@@ -964,7 +964,12 @@ public static function getLegancyConfigSettings(): array
             'components' => [
                 'formatterApp' => new UnsetArrayValue(),
                 'search' => new UnsetArrayValue(),
+                'formatter' => [
+                    'defaultTimeZone' => new UnsetArrayValue(),
+                ],
             ],
+            'timeZone' => new UnsetArrayValue(),
```

Comment:
@yurabakhtin No need to add the TZ stuff here.

Simply add here, then it's removed from existing DynamicConfigs as well: 
https://github.com/humhub/humhub/blob/master/protected/humhub/libs/DynamicConfig.php#L136-L142

Reply:
@luke- Commit https://github.com/humhub/humhub/pull/7243/commits/22dc26cc54f5103864bf25385aec6e117aa97b18.